.github/renovate.json5

```diff
@@ -10,7 +10,7 @@
       // Automerge patches, pin changes and digest changes.
       // Also groups these changes together.
       groupName: "bugfixes",
-      excludePackagePrefixes: ["dev", "lint", "types"],
+      excludePackagePrefixes: ["lint", "types"],
```

Comment:
Excluding dev here made `requirements.txt` and `requirements-dev.txt` get out of sync sometimes